_sass/layout.scss

```diff
@@ -378,7 +378,7 @@ hr {
     line-height: 25px;
     width: 90%;
     font-size: 15px;
-    white-space: pre-line;
```

Comment:
This causes:
![Screenshot from 2023-10-08 03-00-33](https://github.com/meshery/meshery.io/assets/110674407/b12b648e-3f1a-4974-9601-b77f9b5d4fcb)